src/components/sortable-list/sortable-list.tsx

```diff
@@ -165,49 +161,52 @@ export class SortableList implements InteractiveComponent {
 
     this.items = Array.from(this.el.children);
 
-    handle.activated = true;
-    handle.setFocus();
     this.beginObserving();
+    requestAnimationFrame(() => focusElement(handle));
+
+    if ("activated" in handle) {
+      handle.activated = true;
```

Comment:
since handleSelector can be anything we need to do this check. Maybe we shouldn't support a handleSelector if we want to maintain drag consistency thoughout components and require use of a calcite-handle.